airbyte-webapp/package.json

```diff
@@ -4,11 +4,12 @@
   "private": true,
   "scripts": {
     "start": "react-scripts start",
-    "build": "react-scripts build",
+    "build": "node ./license-check && react-scripts build",
```

Comment:
Is it actually a good idea to check it during the build? Shouldn't we check it as a separate CI step instead?

@jrhizor are we currently checking licenses for connectors and backend? If yes - do we check it before build/ as separate step / during the build.

Reply:
I think a separate CI step would make sense for it. But happy to get @jrhizor input for that.

Reply:
It'd probably make sense to add the license check as a NpmTask in https://github.com/airbytehq/airbyte/blob/master/airbyte-webapp/build.gradle that is attached to the `verify` step. Then it wouldn't be part of the `npm build` step, just part of `./gradlew build` / `./gradlew verify`.